lms/djangoapps/courseware/views.py

```diff
@@ -1419,3 +1426,180 @@ def render_xblock(request, usage_key_string, check_if_enrolled=True):
             'xqa_server': settings.FEATURES.get('XQA_SERVER', 'http://your_xqa_server.com'),
         }
         return render_to_response('courseware/courseware-chromeless.html', context)
+
+
+def get_analytics_answer_dist(request):
+    """
+    Calls the the analytics answer distribution api client. Retrieves answer distribution data for the in-line
+    analytics display.
+
+    Arguments:
+        request (django request object):  the HTTP request object that triggered this view function
+
+    Returns:
+        (django response object):  JSON response:
+            500 if error occurred,
+            404 if api client returns no data,
+            otherwise 200 with JSON body.
+    """
+
+    all_data = json.loads(request.POST['data'])
+    module_id = all_data['module_id']
+    question_types_by_part = all_data['question_types_by_part']
+    num_options_by_part = all_data['num_options_by_part']
+    course_key = SlashSeparatedCourseKey.from_string(all_data['course_id'])
+
+    # Construct an error message
+    if getattr(settings, 'ZENDESK_URL'):
+        link_start = "<a href=\"" + getattr(settings, 'ZENDESK_URL') + "/hc/en-us/requests/new\">"
+        link_text = _('here')
```

Comment:
Why don't you just include the text directly? I don't see the point in defining "link_text" as a variable especially when it's going to mess up a ton of translations.